### DISCUSSION TITLE
Infinity 新标签页 (Pro) 改造使用vimium-c记录

### DISCUSSION THREAD

**Discussion by yangfeng20** [gdh1995/vimium-c#1130] [Q&A] [answered]
记录一下过程
https://github.com/gdh1995/vimium-c/wiki/Inject-into-other-extensions
1. Modify HTML pages of the extension   【修改newtable html】
    a. 进入C:\Users\用户名\AppData\Local\Microsoft\Edge\User Data\Default\Extensions\hajlmbnnniemimmaehcefkamdadpjlaa\10.0.129_0\newtab
    b. 修改index.html
    
3. Modify extension properties (only required on Chrome)  【修改manifest，删除_metadata 】
4. Load the modified extension and enable injection 【加载本地插件】
5. 进入Vimium C选项页面，在高级选项中将ID添加到“其他扩展ID白名单”中，然后保存更改

**Comment by gdh1995** [comment, marked as answer]
Congratulations
